workbench/static/workbench/js/runtime/logger.js

```diff
@@ -37,7 +40,7 @@ XBlock generated them and other XBlock-local metadata.
             sendRequest, has;
 
         sendRequest = function(data, options) {
-	    console.log(data);
+	    console.log(JSON.stringify(data));
```

Comment:
In this case, the interactive form was a little bit of a disaster for several reasons. The events are fairly small, but sometimes a little deep. Lots of clicks to see what's going on. In addition, it's nice to be able to cut-and-paste them, pretty print them, compare to edx-platform, paste into Python, etc. The string form lets us do that. I removed a redundant stringify (which was needed for AJAX, but not for console).